pkg/snapshot/filesystem.go

```diff
@@ -765,7 +794,6 @@ func fileSystemMinioConfigPod(clientset kubernetes.Interface, deployOptions File
 						{
 							Name:      "fs",
 							MountPath: "/fs",
-							ReadOnly:  readOnly,
```

Comment:
can we keep this if it's still using minio?

Reply:
I think it's a value add to be able to check if the volume is writable even in the Minio case. I've run afoul of this many times testing, and now the user will be notified in the UI. The check will fail with this flag enabled. 

I can revert depending on how strongly you feel.